src/utils/exerciseServer.ts

```diff
@@ -399,7 +406,9 @@ async function exerciseServerWorker(testDir: string, tsserverPath: string, repla
         }
 
         const replayString = JSON.stringify(request) + "\n";
+        const rawErrorsString = JSON.stringify(request) + "\n";
         await replayScriptHandle.write(replayString);
+        await rawErrorsScriptHandle.write(rawErrorsString);
```

Comment:
I'm missing something.  This appears to be writing the request string to the raw errors file, but the sample run you shared had the call stack from the response.  Is there some reason this won't be exactly the same text as the replay script?

Reply:
You're right. I added this part initially to check if raw error file was being created and yes now the file has call stack from the response. I've removed it now.